mmv1/third_party/terraform/resources/resource_container_node_pool.go.erb

```diff
@@ -79,6 +79,51 @@ func resourceContainerNodePool() *schema.Resource {
 	}
 }
 
+var schemaBlueGreenSettings = &schema.Schema{
+	Type:     schema.TypeList,
+	Optional: true,
+	Computed: true,
+	MaxItems: 1,
+	Elem: &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"standard_rollout_policy": {
+				Type:        schema.TypeList,
+				Required:    true,
+				MaxItems:    1,
+				Description: `Standard rollout policy is the default policy for blue-green.`,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"batch_percentage": {
+							Type:         schema.TypeFloat,
+							Optional:     true,
+							Description:  `Percentage of the blue pool nodes to drain in a batch.`,
+							ValidateFunc: validation.FloatBetween(0.0, 1.0),
+						},
+						"batch_node_count": {
+							Type:        schema.TypeInt,
+							Optional:    true,
+							Description: `Number of blue nodes to drain in a batch.`,
+						},
+						"batch_soak_duration": {
+							Type:        schema.TypeString,
+							Optional:    true,
+							Default:     "0s",
```

Comment:
Also some thoughts on default values. Are these default values fixed and won't be changed anytime in the future? If not, we could have `optional` + `compute` instead, as changing default values is a breaking change in Terraform.